### ISSUE TITLE
Crash upon creating Skygrid World

### ISSUE THREAD

**Issue by AggressivePear** [Funwayguy/SkyGrid8#5]
https://pastebin.com/7aa8qjAm is the error I get. I have changed some of the weights in the config file, but haven't touched much of anything else.

**Comment by Funwayguy** [comment]
did you happen to set any of the weights to zero?

**Comment by AggressivePear** [comment]
Yes. Should I remove the blocks from the config rather than set it to 0?

**Comment by AggressivePear** [comment]
One thing I do need to ask though, why are leaves spawning with a meta value of 8? Does that allow random leaf types?

**Comment by Funwayguy** [comment]
Weights need to be at minimum set to 1 or they break (forgot to add in handling just in case). Just remove the entries you don't want to use.

The leaf metadata is set to that because that should be the non-decaying variant. Otherwise they'd all disappear soon after generating and it would be impossible to harvest saplings

**Comment by AggressivePear** [comment]
I have removed all blocks with the weight of 0, and made sure the last block did not end in a comma, but still the world crashes.  I uploaded the contents of my config files, and the latest crash file. Have a good new year.

[skygrid.zip](https://github.com/Funwayguy/SkyGrid8/files/2717696/skygrid.zip)
[crash-2018-12-30_19.09.05-server.txt](https://github.com/Funwayguy/SkyGrid8/files/2717697/crash-2018-12-30_19.09.05-server.txt)

**Comment by AggressivePear** [comment]
I have found that two of the block entries ended in a double }, and this caused the crash. It works now.

**Comment by Funwayguy** [comment]
I assume the email you just sent me was before this comment was made?

Sorry I didn't get back to you sooner, had my hands a bit full with Better Questing and I keep forgetting to check issues on other mods. I did run your configs and actually didn't notice the maformed JSON error in the log until you pointed it out. Figured it was something I missed in my code up until now.

PS: Comments on issue threads send me notification emails too so you don't actually need to do both.
